jazz-ui/src/app/secondary-components/create-service/internal/create-service.component.html

```diff
@@ -0,0 +1,704 @@
+                                        <div class="input-wrapper">
+                                            <input name="rateDayOfWeek" type="text" class="textForm" [(ngModel)]="cronObj.dayOfWeek" #rateDayOfWeek='ngModel' (ngModelChange)="generateExpression(rateExpression)" required>
+                                        </div>
+                                    </div>
+                                    <div class="col-lg-2">
+                                        <p class="sm-input-label">Year</p>
+                                        <div class="input-wrapper">
+                                            <input name="rateYear" type="text" class="textForm" required [(ngModel)]="cronObj.year" #rateYear='ngModel' (ngModelChange)="generateExpression(rateExpression)">
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="rate-field row  zero-margin arrow_box" *ngIf="rateExpression.type == 'rate'">
+                                    <div class="col-lg-7 padding-right-0">
+                                        <div class="input-wrapper" [ngClass]="{'available': rateField.valid && (rateField.dirty || rateField.touched), 'unavailable': !rateField.valid && (rateField.dirty || rateField.touched)}">
+                                            <input name="rateField" type="number" class="textForm" required [(ngModel)]="rateExpression.duration" (ngModelChange)="generateExpression(rateExpression)" #rateField='ngModel' OnlyNumber="true">
+                                            <img class="input-loader ng-hide">
+                                        </div>
+                                    </div>
+                                    <div class="col-lg-5">
+                                        <div class="input-wrapper">
+                                            <dropdown [dropdwnContent]="rateData" title="Select interval" [selected]="rateExpression.interval" [statusFilter]="statusFilter" (onSelected)="onSelectedDr($event)"></dropdown>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                            <div class="form-error" *ngIf="rateExpression.error != undefined">
+                                {{rateExpression.error}}
+                            </div>
+                            <p *ngIf="rateExpression.type != 'none'" class="learn-more"><a href="https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/ScheduledEvents.html">Learn more</a> about CloudWatch Event schedules</p>
+                        </div>
+                    </div>
+                </div>
+            </section>
+            <!-- start of aws events -->
+            <section class="each-step-wrap aws-events" *ngIf="typeOfService == 'function'">
```

Comment:
this needs to be removed for OSS.

Reply:
this feature is for internal only
for OSS /create-service/oss/create-service.component.html component is pickedup during the build